examples/connectivity/plot_extract_regions_canica_maps.py

```diff
@@ -0,0 +1,150 @@
+                         title='ICA components')
+
+################################################################################
+# Extracting regions from ICA maps and then timeseries signals from those
+# regions, both can be done by importing Region Extractor from regions module.
+# threshold=0.5 indicates that we keep nominal of amount nonzero voxels across all
+# maps, less the threshold means that more intense non-voxels will be survived.
+from nilearn.regions import RegionExtractor
+
+extractor = RegionExtractor(components_img, threshold=0.5,
+                            thresholding_strategy='ratio_n_voxels',
+                            extractor='local_regions',
+                            standardize=True, min_region_size=1350)
+# Just call fit() to process for regions extraction
+extractor.fit()
+# Extracted regions are stored in regions_img_
+regions_extracted_img = extractor.regions_img_
+# Each region index is stored in index_
+regions_index = extractor.index_
+# Total number of regions extracted
+n_regions_extracted = regions_extracted_img.shape[-1]
+
+# Visualization
+# Show region extraction results
+title = ('%d regions are extracted from %d ICA components.'
+         '\nEach separate color of region indicates extracted region'
+         % (n_regions_extracted, 5))
+plotting.plot_prob_atlas(regions_extracted_img, view_type='filled_contours',
+                         title=title)
+
+################################################################################
+# Computing correlation coefficients
+# First we need to do subjects timeseries signals extraction and then estimating
+# correlation matrices on those signals.
+# To extract timeseries signals, we call transform() from RegionExtractor object
+# onto each subject functional data stored in func_filenames.
+# To estimate correlation matrices we import connectome utilities from nilearn
+from nilearn.connectome import ConnectivityMeasure
+
+correlations = []
+# Initializing ConnectivityMeasure object with kind='correlation'
+connectome_measure = ConnectivityMeasure(kind='correlation')
+for filename, confound in zip(func_filenames, confounds):
+    # call transform from RegionExtractor object to extract timeseries signals
+    timeseries_each_subject = extractor.transform(filename, confounds=confound)
+    # call fit_transform from ConnectivityMeasure object
+    correlation = connectome_measure.fit_transform([timeseries_each_subject])
+    # saving each subject correlation to correlations
+    correlations.append(correlation)
+
+# Mean of all correlations
+import numpy as np
+
+mean_correlations = np.mean(correlations, axis=0).reshape(n_regions_extracted,
+                                                          n_regions_extracted)
+
+# Visualization
+# Showing mean correlation results
+# Import image utilities in utilising to operate on 4th dimension
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
I think that this import can be avoided by relying on nilearn.plotting utilities.